dependencies/dex/config.yaml

```diff
@@ -0,0 +1,33 @@
+issuer: https://localhost:9443/idp/
+storage:
+  type: kubernetes
+  config:
+    inCluster: true
+web:
+  https: 0.0.0.0:9443
+  tlsCert: /etc/dex/tls/tls.crt
+  tlsKey: /etc/dex/tls/tls.key
+connectors:
+- type: mockCallback
```

Comment:
What is this for?

Reply:
It allows bypassing the authentication process. I'll remove it.